setup.py

```diff
@@ -8,21 +8,23 @@
     author_email='solutions@caktusgroup.com',
     packages=find_packages(),
     include_package_data=True,
-    url='http://github.com/caktus/rapidsms-decisiontree-app/',
+    url='https://github.com/caktus/rapidsms-decisiontree-app/',
     license='BSD',
-    description='RapidSMS Decisiontree',
+    description=" ".join(__import__('decisiontree').__doc__.splitlines()).strip(),
     long_description=open('README.rst').read(),
     classifiers=(
         'Topic :: Internet :: WWW/HTTP :: Dynamic Content',
         'Intended Audience :: Developers',
         'License :: OSI Approved :: BSD License',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2.6',
```

Comment:
Looks like this can stay, right? (Though I'd be fine with dropping it if we need to)